packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -136,6 +136,16 @@ class AnnotationResult<T> {
 ///  * [RenderView.compositeFrame], which implements this recomposition protocol
 ///    for painting [RenderObject] trees on the display.
 abstract class Layer extends AbstractNode with DiagnosticableTreeMixin {
+  final Map<int, VoidCallback> _callbacks = <int, VoidCallback>{};
+
+  void _fireCompositionCallbacks(bool includeChildren) {
```

Comment:
nit: make the argument a required optional parameter to make callsites easier to read.

Reply:
Done